controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -345,9 +345,23 @@ func (r *OCSInitializationReconciler) ensureOcsOperatorConfigExists(initialData
 			Name:      util.OcsOperatorConfigName,
 			Namespace: initialData.Namespace,
 		},
-		Data: ocsOperatorConfigData,
```

Comment:
I guess the data section was there do that when the CM is created the data section is not null, will removing this not create any issue?

Reply:
No, it does not create any issues, I have tested it.